clang/include/clang/Driver/Options.td

```diff
@@ -6877,6 +6877,11 @@ defm sycl_esimd_force_stateless_mem : BoolFOption<"sycl-esimd-force-stateless-me
   NegFlag<SetFalse, [], [ClangOption, CLOption], "Do not enforce using "
           "stateless memory accesses.">,
   BothFlags<[], [ClangOption, CLOption, CC1Option], "">>;
+defm sycl_rtc_mode: BoolFOption<"sycl-rtc-mode",
+  LangOpts<"SYCLRTCMode">, DefaultFalse,
+  PosFlag<SetTrue, [], [ClangOption], "RTC Mode On">,
+  NegFlag<SetFalse, [], [ClangOption], "RTC Mode Off">,
```

Comment:
```suggestion
  PosFlag<SetTrue, [], [ClangOption], "Enable">,
  NegFlag<SetFalse, [], [ClangOption], "Disable">,
```
"RTC Mode On RTC mode in SYCL." didn't make much sense to me :)

Reply:
Good catch, thanks!